### ISSUE TITLE
gzip not working

### ISSUE THREAD

**Issue by BruceCham** [indexzero/http-server#471]
## gzip===true Why not default conversion to gzip format?
```js
// ecstatic.js p 421
 // serve gzip file if exists and is valid
    function tryServeWithGzip() {
      fs.stat(gzippedFile, (err, stat) => {
        if (!err && stat.isFile()) {
          hasGzipId12(gzippedFile, (gzipErr, isGzip) => {
            if (!gzipErr && isGzip) {
              file = gzippedFile;
              serve(stat);
            } else {
              statFile();
            }
          });
        } else {
          statFile();
        }
      });
    }
```

**Comment by thornjad** [comment]
Are you asking why gzip is disabled by default, or are you reporting that gzip is not working when enabled?

**Comment by BruceCham** [comment]
Automatically convert files to gzip format and return them when enabled.

**Comment by BigBlueHat** [comment]
> Automatically convert files to gzip format and return them when enabled.

@BruceCham neither `http-server` nor the underlying `node-ecstatic` projects do this currently, and (afaict) are unlikely too as it would step away from the "simple static server" mantra we've practiced so far.

Closing the issue as "wontfix," but do let me know if you're wanting something different, though, and we can discuss it further and/or reopen.

Cheers!
